builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -838,7 +838,12 @@ def updatePolicy(env, destARN){
 
 def writeServerlessFile(config, env, dataObjValues) {
 
-  def iamRoleArnValue = dataObjValues.IAM.USERSERVICES_ROLEID;
+  def iamRoleArnValue;
+  if(config['domain'] == 'jazz'){
+      iamRoleArnValue = config['iamRoleARN']
+  } else {
+      iamRoleArnValue = dataObjValues.IAM.USERSERVICES_ROLEID;
```

Comment:
"dataObjValues" Please use meaningful names

Reply:
changes the name and pushed